packages/neuron-ui/src/components/SUDTSend/hooks.ts

```diff
@@ -152,6 +160,26 @@ export function getGenerator(sendType?: SendType) {
   return generateSUDTTransaction
 }
 
+export async function batchGenerateExperimental(
+  experimental: { tx: any; assetAccount?: any; params?: any },
+  priceArray: string[]
+) {
+  if (experimental?.params) {
+    const generator = getGenerator(experimental?.params?.sendType)
+    const requestArray = priceArray.map(itemPrice => generator({ ...experimental.params, feeRate: itemPrice }))
+    const allPromiseResult = await Promise.allSettled(requestArray)
+    const resList = (allPromiseResult as GenerateResult[]).map((batchItem, index: number) => ({
```

Comment:
`as` is like `ts-ignore` and `any`, all of which break the type system.
This is not better than `ts-ignore`, because at least `ts-ignore` clearly indicates that the type system is being broken.
Please refer to this approach: https://github.com/nervosnetwork/neuron/pull/2765#discussion_r1257826045 .